debian/ubuntu-advantage-tools.postinst

```diff
@@ -359,6 +364,37 @@ migrate_user_config_post() {
     fi
 }
 
+rename_gpg_keys() {
+    # esm-infra
+    if [ -f "$APT_TRUSTED_KEY_DIR/$OLD_ESM_INFRA_KEY_TRUSTY" ]; then
+        mv "$APT_TRUSTED_KEY_DIR/$OLD_ESM_INFRA_KEY_TRUSTY" "$APT_TRUSTED_KEY_DIR/$ESM_INFRA_KEY"
+    fi
+    # esm-apps
+    if [ -f "$APT_TRUSTED_KEY_DIR/$OLD_ESM_APPS_KEY" ]; then
+        mv "$APT_TRUSTED_KEY_DIR/$OLD_ESM_APPS_KEY" "$APT_TRUSTED_KEY_DIR/$ESM_APPS_KEY"
+    fi
+    # cc-eal
+    if [ -f "$APT_TRUSTED_KEY_DIR/ubuntu-advantage-cc-eal.gpg" ]; then
+        mv "$APT_TRUSTED_KEY_DIR/ubuntu-advantage-cc-eal.gpg" "$APT_TRUSTED_KEY_DIR/ubuntu-pro-cc-eal.gpg"
+    fi
+    # cis (and usg)
+    if [ -f "$APT_TRUSTED_KEY_DIR/ubuntu-advantage-cis.gpg" ]; then
+        mv "$APT_TRUSTED_KEY_DIR/ubuntu-advantage-cis.gpg" "$APT_TRUSTED_KEY_DIR/ubuntu-pro-cis.gpg"
+    fi
+    # fips (+upodates)
```

Comment:
```suggestion
    # fips (+updates)
```
:upside_down_face: 
and copypastad below

Reply:
didn't you know about this? we decided it will be called `upodates` from next version.
I will keep you upodated.